server/core/runtime/models/shell_command_runner.go

```diff
@@ -0,0 +1,155 @@
+package models
```

Comment:
I don't think this is a particularly good place for this, but noticed it was where the other `exec` functionality was put. Not totally sure on the naming convention of this project, but it couldn't be in `runtime` because of import cycles with the `terraform` package.